core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/WorkflowWorker.scala

```diff
@@ -126,14 +129,39 @@ class WorkflowWorker(
     }
   }
 
+  override def preStart(): Unit = {
+    val config = workFlowReporter.readConfiguration(Kamon.config())
+    logger.info(s"""Start workflow worker reporter:${self.path.toString}""")
+    var reporter = new workFlowReporter(
```

Comment:
Why is this reporter defined locally? I thought it should be bond to the worker?